libs/@guardian/source/src/react-components/icons/icons.ts

```diff
@@ -22,12 +22,12 @@ export { SvgAsterisk } from '../__generated__/icons/SvgAsterisk';
 export { SvgAudio } from '../__generated__/icons/SvgAudio';
 export { SvgAudioMute } from '../__generated__/icons/SvgAudioMute';
 export { SvgBin } from '../__generated__/icons/SvgBin';
-export { SvgBookOutlined } from '../__generated__/icons/SvgBookOutlined';
 export { SvgBookmarkCross } from '../__generated__/icons/SvgBookmarkCross';
 export { SvgBookmarkFilled } from '../__generated__/icons/SvgBookmarkFilled';
 export { SvgBookmarkOutlined } from '../__generated__/icons/SvgBookmarkOutlined';
 export { SvgBookmarkRoundFilled } from '../__generated__/icons/SvgBookmarkRoundFilled';
 export { SvgBookmarkRoundOutlined } from '../__generated__/icons/SvgBookmarkRoundOutlined';
+export { SvgBookOutlined } from '../__generated__/icons/SvgBookOutlined';
```

Comment:
I think you also need to export the new icons from `index.ts`? Maybe you could just re export this file

Reply:
Good spot! Have updated.